src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -333,29 +333,29 @@ The following workflows represent some mathematical functions that can be optimi
 
 *Himmelblau_Function:* is a multi-modal function, used to test the performance of optimization algorithms.
 
-== Machine Learning Model as a Service (ML MaaS)
+== Machine Learning Model as a Service (MLaaS)
```

Comment:
Should we keep Model? `MLaas` should refer to `Machine Learning as a Service` I think. To verify with Denis.

Reply:
I fixed it.
Thank you @mklkun